Harvest.Web/ClientApp/src/Projects/AdhocProject.tsx

```diff
@@ -58,6 +60,8 @@ export const AdhocProject = () => {
   const [accounts, setAccounts] = useState<ProjectAccount[]>([]);
   const [disabled, setDisabled] = useState<boolean>(true);
 
+  const [quote, setQuote] = useState<QuoteContent>(new QuoteContentImpl());
```

Comment:
This should not be a state variable -- you aren't ever calling setQuote() and react doesn't need to know about the content.